Cartfile.resolved

```diff
@@ -1,4 +1,5 @@
-github "ReactiveCocoa/ReactiveCocoa" "7.0.0"
-github "ReactiveCocoa/ReactiveSwift" "3.0.0"
-github "Vadim-Yelagin/DataSource" "0.5.2"
+github "Fueled/DataSource" "0.5.4"
+github "Fueled/ios-utilities" "1.4"
+github "ReactiveCocoa/ReactiveCocoa" "7.2.0"
+github "ReactiveCocoa/ReactiveSwift" "3.1.0"
```

Comment:
Weird that the dependencies are resolved as `7.X`, `ReactiveCocoa` should be on `8.0` 🤔 Did you run `carthage update --use-submodules --no-build`?

Reply:
Yes, I did it. But I had an old version of Carthage (0.26.2) and that was the reason why Carthage resolved with an old version of libraries (7.X). I've updated Carthage to the latest version (0.31.2) and now Carthage gets the correct version (8.X) of modules. Thanks for pointing that out 👍

Reply:
But after that Carthage is fetching old version of DataSource :-)
```
Alekseis-MacBook-Pro:DataSourceExample alekseibobrov$ carthage update --use-submodules --no-build
*** Fetching DataSource
*** Fetching ios-utilities
*** Fetching ReactiveCocoa
*** Fetching ReactiveSwift
*** Fetching Result
*** Checking out ReactiveSwift at "4.0.0"
*** Checking out Result at "4.1.0"
*** Checking out ios-utilities at "1.5"
*** Checking out DataSource at "0.2.6"
*** Checking out ReactiveCocoa at "8.0.2"
Alekseis-MacBook-Pro:DataSourceExample alekseibobrov$ 

```

Reply:
Yes, i did it. We see 7.X version of 7.X ReactiveCocoa because of the Fueled/DataSource last release version is 0.5.4 which is using 7.X Reactive Cocoa. I think that is the reason: Fueled/DataSource latest version is 0.5.4 (similar to Vadim latest version) which is using 7.X ReactiveCocoa. 

I tried to point to your last commit (which include the latest version of ReactiveCocao) in the Cartfile: 

github "Fueled/DataSource" #55085c7cda4f539b950463742476afed6212a43e
github "Fueled/ios-utilities"

And, since your last commit is using 8.X ReactiveCocoa we get the right version of ReactiveCocoa. It's expected to be that.  Everything should be fine, but we got another problem: Carthage fetch the wrong version of DataSource. somehow Carthage fetches 0.2.6 version of DataSource instead of 0.5.4. Here is the log:

```
Alekseis-MacBook-Pro:DataSourceExample alekseibobrov$ carthage update --use-submodules --no-build
*** Fetching DataSource
*** Fetching ios-utilities
*** Fetching ReactiveCocoa
*** Fetching ReactiveSwift
*** Fetching Result
*** Checking out Result at "4.1.0"
*** Checking out ios-utilities at "1.5"
*** Checking out ReactiveSwift at "4.0.0"
_*** Checking out DataSource at "0.2.6"_
*** Checking out ReactiveCocoa at "8.0.2"
```

Looks like it's a problem with a tag on the project. Maybe somehow it's set to 0.2.6 for your last commit that I am using. Can you please check this out?

Reply:
@aleksei-bobrov you're right indeed, I've just pushed the tag `0.5.5` that has the updates, so you can run carthage update here and have it refer to 8.0 😉

Reply:
@stephanecopin 
I just checked and nothing has changed: 

```
Alekseis-MacBook-Pro:DataSourceExample alekseibobrov$ carthage update --use-submodules --no-build
*** Fetching DataSource
*** Fetching ios-utilities
*** Fetching ReactiveCocoa
*** Fetching ReactiveSwift
*** Fetching Result
*** Checking out DataSource at "0.2.6"
*** Checking out ReactiveSwift at "4.0.0"
*** Checking out ios-utilities at "1.5"
*** Checking out Result at "4.1.0"
*** Checking out ReactiveCocoa at "8.0.2"

```
Maybe we need to add a release tag a release section of a GitHub project?  https://github.com/Fueled/DataSource/releases

Reply:
@aleksei-bobrov seems like the tag didn't push, I've just verified and it should be good to go now, here's the output I got when trying it locally:
```bash
*** Cloning ios-utilities
*** Cloning DataSource
*** Fetching ReactiveCocoa
*** Fetching ReactiveSwift
*** Fetching Result
*** Checking out ios-utilities at "1.5"
*** Checking out ReactiveCocoa at "8.0.2"
*** Checking out ReactiveSwift at "4.0.0"
*** Checking out DataSource at "0.5.5"
*** Checking out Result at "4.1.0"
```